modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java

```diff
@@ -29,18 +29,19 @@
  */
 public class ObjectDefinitionTestUtil {
 
-	public static ObjectDefinition addAndPublishCustomObjectDefinition()
+	public static ObjectDefinition addAndPublishCustomObjectDefinition(
+			boolean enableLocalization, List<ObjectField> objectFields)
```

Comment:
@p-albuquerque you could add a new method without the `List<ObjectField> objectFields` parameter

```
public static ObjectDefinition publishObjectDefinition(boolean enableLocalization) {
    return publishObjectDefinition(enableLocalization, Collections.emptyList());
}
```